src/patternfly/sass-utilities/placeholders.scss

```diff
@@ -6,6 +7,10 @@
   --pf-global--link--Color: var(--pf-global--link--Color--dark);
   --pf-global--link--Color--hover: var(--pf-global--link--Color--dark--hover);
   --pf-global--BackgroundColor--100: var(--pf-global--BackgroundColor--light-100);
+
+  .pf-c-divider {
+    --pf-c-divider--after--BackgroundColor: var(--pf-global--BorderColor--100);
```

Comment:
I'd love to remove all of the component specific stuff from these blocks if we can. This is going to change the color on any divider in a `pf-t-dark` block, but leave `BorderColor--100` alone. Dividers in the new dark theme are closer to this, but that comes from `--pf-global--BorderColor--100`, because (as far as I know from design) dividers are a single line border using the border color. 

So using `.pf-t-dark` we'll get something like this in tabs, where a divider wouldn't match the tabs bottom border

<img width="897" alt="Screenshot 2023-01-17 at 7 05 10 PM" src="https://user-images.githubusercontent.com/35148959/213055815-8733788b-5b2d-4bbd-8011-ced86f77ec59.png">

This is what it looks like currently

<img width="881" alt="Screenshot 2023-01-17 at 7 04 27 PM" src="https://user-images.githubusercontent.com/35148959/213055881-ad7f3202-0cd0-4a51-9a73-524539e9e1ef.png">

and fwiw, this is what it looks like in the new dark theme

<img width="884" alt="Screenshot 2023-01-17 at 7 08 19 PM" src="https://user-images.githubusercontent.com/35148959/213055981-ea30a206-f662-4d74-a723-9a0a7be2abd1.png">

I guess it's worth considering if we should update `--pf-global--BorderColor--100` here instead of creating an exception.